encodings/protobuf/README.md

```diff
@@ -0,0 +1,178 @@
+# OCSF Protobuf Encoding
+
+This directory contains a sample Protobuf encoding for OCSF.
+
+Although encoding in general is _informative_ in OCSF, enabling the
+interoperation between systems in a single organization and indeed
+between organizations indicates that a **canonical** protobuf encoding
+has significant utility.  This directory contains that canonical
+encoding.
+
+## Semantic Type Considerations
+
+### Primitive Types
+
+The primitive types are mapped as follows
+
+| OCSF Type   | Protobuf Type           |
+| ----------- | ----------------------- |
+| `boolean_t` | `bool`                  |
+| `float_t`   | `float`                 |
```

Comment:
I think you mean `double` for the Protobuf type based on your [go code](https://bitbucket.org/dkolbly/osf-meta/src/df66e7443594bb9c2ac4b424ab84ccd134c7bfa7/cmd/osf-sdk-protobuf/sdkprotobuf.go#lines-727) and the generated proto.

I think `double` is the correct choice for what it's worth.